code/frameworks/nextjs/README.md

```diff
@@ -159,12 +159,16 @@ export default {
   framework: {
     name: '@storybook/nextjs',
     options: {
+      image: {
+        loading: 'eager',
+      },
       nextConfigPath: path.resolve(__dirname, '../next.config.js'),
     },
   },
 };
 ```
 
+- `image`: Props to pass to every instance of `next/image`
```

Comment:
This pull request was originally supposed to just allow toggling lazy loading for every instance of `next/image` using parameters, but I thought we might as well allow setting any property.